lib/galaxy/model/__init__.py

```diff
@@ -5304,3 +4813,55 @@ def copy_list(lst, *args, **kwds):
         return lst
     else:
         return [el.copy(*args, **kwds) for el in lst]
+
+
+# Sample request associated class
+class Request(object, Dictifiable):
```

Comment:
Out of curiosity, can we actually remove these, or does it break things if they don't exist?

Reply:
These classes are used in the model mapping.

Reply:
Got an example?  Can't we remove them from the mapping, too?

Reply:
The `HistoryDatasetAssociationAnnotationAssociation` are defined in a similar fashion. I removed the mapping and model too without modifying the migration scripts, fyi. The alternative would be to leave the mapping as it is.